content/altinn-studio/guides/development/signing/api-signing/_index.en.md

```diff
@@ -0,0 +1,110 @@
+---
+title: API based signing
+linktitle: API based signing
+description: Follow these steps to implement API based signing in your app.
+tags: [signering]
+weight: 53
+aliases:
+- /nb/altinn-studio/guides/signing/api-signing
+---
+
+## What do we mean by API based signing?
+
+{{% insert "content/altinn-studio/guides/development/signing/api-signing/intro.en.md" %}}
+
+## Prerequisites
+
+If the service should send a signature receipt to the inbox of the signee, the Altinn message service (Correspondence) must be enabled.
+
+## Configuring signing
+
+You can see how to configure signing in an app using the guides found [here](/altinn-studio/guides/development/signing/).
+API based signing has the same requirements for configuration, except the layout-set for the signing step can be simpler.
+
+## The most important API calls
+
+### Perform single signing
+If only one person should sign in a signing step, you can use the "process next" endpoint:
+
+`PUT /{org}/{app}/instances/{instanceOwnerPartyId}/{instanceGuid}/process/next`
+  
+Body:
+```json
+{
+  "action": "sign"
+}
+```
+
+In this case, the process will automatically navigate to the next step once the signature has been saved.
+If more people should sign in sequence after each other, more signature steps can be added in a chain.
```

Comment:
Er "in sequence after each other" litt smør på flesk? 😄 og "in a chain" er på en måte det samme, men sagt på en annen måte igjen.